src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
+  {
+    if (*val != 0)
```

Comment:
In addition, just to check that everything is covered... where the parsing error for create/update subscriptions (where q is in the payload) in the case of wrong q expression?

(Maybe it is self-evident in the PR, but it is a bit long and a hint will be great to locate that piece of code :)